contentcuration/contentcuration/frontend/channelEdit/vuex/contentNode/actions.js

```diff
@@ -178,6 +178,20 @@ export function addNextStepToNode(context, { targetId, nextStepId }) {
 /* CONTENTNODE EDITOR ACTIONS */
 export function createContentNode(context, { parent, kind = 'topic', ...payload }) {
   const session = context.rootState.session;
+
+  const channel = context.rootGetters['currentChannel/currentChannel'];
+  let contentDefaults = Object.assign({}, channel.content_defaults);
+
+  // content_defaults for historical reason has stored the license as a string constant,
+  // but the serializers and frontend now use the license ID. So make sure that we pass
+  // a license ID when we create the content node.
+  if (kind === 'topic') {
+    // Don't assign a license to topics, even if one is set in content_defaults
+    contentDefaults.license = null;
```

Comment:
Could you also make license_description, author, aggregator, provider, and copyright_holder null here?